mlflow/utils/annotations.py

```diff
@@ -21,15 +22,29 @@ def deprecated(alternative=None, since=None):
     :param func: A function to mark
     :returns Decorated function.
     """
-    def deprecated_func(func):
+
+    def deprecated_decorator(func):
         since_str = " since %s" % since if since else ""
-        notice = ".. Warning:: Deprecated%s: This method will be removed in " % since_str + \
-                 "a near future release."
+        notice = (
+            ".. Warning:: {function_name} is deprecated{since_string}. This method will be"
+            " removed in a near future release".format(
+                function_name=func.__name__,
```

Comment:
Can we include the module name with the function? I had left a comment about this in the previous round of reviews, but my suggested solution for correcting the docstring didn't take this into account.

Reply:
Yep, restored